drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -1390,7 +1411,7 @@ void RasterizerStorageGLES3::texture_bind(RID p_texture, uint32_t p_texture_no)
 
 	ERR_FAIL_COND(!texture);
 
-	glActiveTexture(GL_TEXTURE0 + p_texture_no);
+	gl_wrapper.gl_active_texture_max(GL_TEXTURE0 + p_texture_no);
```

Comment:
Maybe this could use some logic to decide whether to apply to the max or minimum, just in case the slot ID is near the end. :thinking: